src/collector/watcher.ts

```diff
@@ -24,6 +24,16 @@ async function detectValidators(txs: Tx[]) {
   // extract validator addresses from string
   const addresses = uniq(
     txs
+      .filter((tx: any) => {
+        const msgs = tx?.value?.msg
+
+        if (!Array.isArray(msgs)) {
+          return false
+        }
+
+        // Ignore oracle tx
+        return msgs.filter((msg) => typeof msg.type === 'string' && msg.type.includes('oracle/')).length === 0
```

Comment:
Here you can use [some](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some), because it will break when find the first msg with type `'oracle/` and returns a boolean.

```js
return msgs.some((msg) => typeof msg.type === 'string' && !msg.type.includes('oracle/'));
```